pkg/ifuzz/powerpc/powerpc.go

```diff
@@ -27,13 +27,15 @@ type InsnBits struct {
 }
 
 type Insn struct {
-	Name   string
-	M64    bool // true if the instruction is 64bit _only_.
-	Priv   bool
-	Pseudo bool
-	Fields map[string][]InsnBits // for ra/rb/rt/si/...
-	Opcode uint32
-	Mask   uint32
+	Name    string
+	Priv    bool
+	Pseudo  bool
+	Fields  map[string][]InsnBits // for ra/rb/rt/si/...
+	Opcode  uint32
+	Mask    uint32
+	Fields2 map[string][]InsnBits
+	Opcode2 uint32
```

Comment:
`PrefixedOpcode` or something like that would be somewhat more explicit as to the purpose of those fields. Same for other similar fields.

Reply:
Well, the Linux kernel uses `suffix`:
```
struct ppc_inst {  
        u32 val;   
#ifdef CONFIG_PPC64
        u32 suffix;
#endif             
} __packed;        
```
so I can make those `FieldsSuffix` and `OpcodeSuffix`, will that be any better?

Reply:
Yes, I think that'd be better :)